<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mapping</class>
 <widget class="QWidget" name="Mapping">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>581</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>782</width>
    <height>581</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,5,0,0,0">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Mapping</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">border-left: 2px solid rgb(170, 85, 127);</string>
            </property>
            <property name="text">
             <string>Ligand A</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">border-left: 1px solid rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Ligand B</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Perturbation State A</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_8">
            <property name="styleSheet">
             <string notr="true">border-top-color: rgb(0, 0, 0);
border-top-width: 2px;</string>
            </property>
            <property name="text">
             <string>Perturbation State B</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGraphicsView" name="graphicsView"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Highlight</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Mutation</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="button_showPertHermite">
            <property name="text">
             <string>Show Pert in Hermite</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_showMapHermite">
            <property name="text">
             <string>Show mapping in Hermite</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Ragen Mapping</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QCheckBox" name="check_mapRegen">
               <property name="text">
                <string>Manuing Mapping</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_editMapp"/>
             </item>
             <item>
              <widget class="QPushButton" name="button_browse_map">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="button_mapRegen">
             <property name="text">
              <string>Regen</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,18">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Pairs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <column>
           <property name="text">
            <string>Ligand A</string>
           </property>
           <property name="textAlignment">
            <set>AlignJustify|AlignVCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ligand B</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>sim. score</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>imgs/ArrowDouble.png</normaloff>imgs/ArrowDouble.png</iconset>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_next_pertGraph">
     <property name="text">
      <string>Confirm Mapping &amp; go to set parameters</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
